hack/godep-save.sh

```diff
@@ -43,6 +38,4 @@ REQUIRED_BINS=(
   "./..."
 )
 
-pushd "${ORIGIN_ROOT}" > /dev/null
```

Comment:
As of this commit, the `hack/lib/init.sh` API will ensure you're in the root after you `source` it so these statements were superfluous.